std/json.d

```diff
@@ -75,7 +75,8 @@ enum JSONOptions
 {
     none,                       /// standard parsing
     specialFloatLiterals = 0x1, /// encode NaN and Inf float values as strings
-    escapeNonAsciiChars = 0x2   /// encode non ascii characters with an unicode escape sequence
+    escapeNonAsciiChars = 0x2,   /// encode non ascii characters with an unicode escape sequence
+    doNotEscapeSlashes = 0x4, /// do not escape slashes ('/')
```

Comment:
Negative options are ungood, but I can't immediately think of a better name. Maybe `rawSlashes`?

Comments got out of alignment BTW